src/query/ee/src/hilbert_clustering/handler.rs

```diff
@@ -98,7 +100,7 @@ impl HilbertClusteringHandler for RealHilbertClusteringHandler {
         }
 
         let rows_per_block =
-            block_thresholds.calc_rows_per_block(checker.total_size, checker.total_rows) as u64;
+            block_thresholds.calc_rows_per_block(checker.total_size, checker.total_rows, 0) as u64;
```

Comment:
Why passes 0 as `calc_rows_per_block` here?

Reply:
The code here will be refactored. calc_rows_per_block takes the maximum of bytes and compressed, so setting it to 0 won't have any effect.